taxi/views.py

```diff
@@ -111,7 +154,7 @@ class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
 def toggle_assign_to_car(request, pk):
     driver = Driver.objects.get(id=request.user.id)
     if (
-        Car.objects.get(id=pk) in driver.cars.all()
+            Car.objects.get(id=pk) in driver.cars.all()
```

Comment:
Consider adding error handling to check if the car with the given `pk` exists before attempting to remove or add it to the driver's cars. This will prevent potential errors if the car does not exist.